### PR TITLE
Add flag to skip AzureWebJobsStorage check.

### PR DESCRIPTION
**Pull request by juliusl**
Addresses #1790

### CODE REVIEW

File: test/Azure.Functions.Cli.Tests/ActionsTests/StartHostActionTests.cs [hunk 1]
```diff
@@ -72,6 +72,29 @@ public async Task CheckNonOptionalSettingsDoesntThrowOnMissingAzureWebJobsStorag
             exception.Should().BeNull();
         }
 
+        [Fact]
+        public async Task CheckNonOptionalSettingsDoesntThrowOnMissingAzureWebJobsStorageWhenFlagIsSet()
+        {
+            var fileSystem = GetFakeFileSystem(new[]
+   {
```
Comment (ahmelsayed) [inline test/Azure.Functions.Cli.Tests/ActionsTests/StartHostActionTests.cs]: nit: formatting

File: test/Azure.Functions.Cli.Tests/ActionsTests/StartHostActionTests.cs [hunk 2]
```diff
@@ -72,6 +72,29 @@ public async Task CheckNonOptionalSettingsDoesntThrowOnMissingAzureWebJobsStorag
...
+                ("x:\\folder1", "{'bindings': [{'type': 'httpTrigger'}]}"),
+                ("x:\\folder2", "{'bindings': [{'type': 'httpTrigger'}]}")
```
Comment (ahmelsayed) [inline test/Azure.Functions.Cli.Tests/ActionsTests/StartHostActionTests.cs]: I think you might wanna change one of these to `timerTrigger` or some other trigger that requires storage check?

Comment (juliusl) [reply test/Azure.Functions.Cli.Tests/ActionsTests/StartHostActionTests.cs]: Good call

**Pull request merged**